htdocs/don/document.php

```diff
@@ -182,7 +182,7 @@
 	print dol_get_fiche_end();
 
 	$modulepart = 'don';
-	$permission = $user->rights->don->lire;
+	$permission = $user->rights->don->creer;
```

Comment:
permission may be to read and permtoedit to edit. So should be lire, shouldn't it ?

Reply:
No, permission is here for a old compatibility and all document file, we have permission = permtoadd = creer and in file /core/tpl/document_actions_post_headers.tpl.php, we have this part : 

![image](https://user-images.githubusercontent.com/2341395/114450215-07bc0000-9bd6-11eb-9813-f752ed1eed41.png)